machine/fdt.c

```diff
@@ -144,6 +144,22 @@ int fdt_string_list_index(const struct fdt_scan_prop *prop, const char *str)
   return -1;
 }
 
+void fdt_version_prop_print(const uint32_t * version_value, int version_len)
+{
+  const char *char_data = (char *)version_value;
```

Comment:
```suggestion
  const char *char_data = (const char *)version_value;
```
since version_value is a `const uint32_t *`